qa/rpc-tests/wallet_nullifiers.py

```diff
@@ -21,7 +21,7 @@ def run_test (self):
         # send node 0 taddr to zaddr to get out of coinbase
         mytaddr = self.nodes[0].getnewaddress();
         recipients = []
-        recipients.append({"address":myzaddr0, "amount":10.0})
+        recipients.append({"address":myzaddr0, "amount":10.00000000-0.00010000}) # utxo amount less fee
```

Comment:
0.0001 isn't exactly representable in IEEE floating point. This works anyway because the decimal printed form of 10.00000000-0.00010000 is exactly 9.9999, but it bothers me that we are using floating point at all. Should this be ``Decimal(10.0) - Decimal(0.0001)``?